api/profile/profileRouter.js

```diff
@@ -381,4 +381,22 @@ router.get('/match/:id', authRequired, (req, res, next) => {
     });
 });
 
+router.post('/mentor-information/', (req, res, next) => {
+  axios
+    .post(`${process.env.DS_API_URL}/read/mentor`, req.body)
```

Comment:
Capturing our conversation about potentially changing these 'getters' as 'gets'.

One thing to consider there for any field qualifiers could live on rout params/queryStrs

Reply:
Code comments added and pushed up to the repo per that conversation. Thank you for the review and the guidance @ryan-hamblin!